SIGSEGV on rewards.GetClaimStatus

https://discord.com/channels/405159462932971535/468923220607762485/1059627314226401290
Report from tk on Discord: 

2023/01/02 05:41:03 Primary execution client is unavailable (Client claims to have finished syncing, but its last block was from 1h9m28.645855368s ago. It likely doesn't have enough peers) and no fallback execution client is configured.
2023/01/02 05:41:03 Error getting deposit pool balance: Could not get deposit pool balance: no Execution clients were ready
2023/01/02 05:41:03 Error getting ETH1 latest reportable block: Could not get latest reportable block: no Execution clients were ready
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x5b6e1d]

goroutine 196899 [running]:
math/big.(*Int).And(0xc00054bab0, 0x0?, 0x51ba71a67e0a3ca5?)
    /usr/local/go/src/math/big/int.go:1080 +0x1d
github.com/rocket-pool/smartnode/shared/services/rewards.GetClaimStatus(0xc000313cb0, {0x59, 0x4a, 0x69, 0xea, 0x90, 0xec, 0xcb, 0xc, 0x91, ...})
    /smartnode/shared/services/rewards/manager.go:66 +0x585
github.com/rocket-pool/smartnode/rocketpool/node/collectors.(*NodeCollector).Collect.func3()
    /smartnode/rocketpool/node/collectors/node-collector.go:233 +0x10b
golang.org/x/sync/errgroup.(*Group).Go.func1()
    /go/pkg/mod/golang.org/x/sync@v0.0.0-20210220032951-036812b2e83c/errgroup/errgroup.go:57 +0x67
created by golang.org/x/sync/errgroup.(*Group).Go
    /go/pkg/mod/golang.org/x/sync@v0.0.0-20210220032951-036812b2e83c/errgroup/errgroup.go:54 +0x8d

--- a/shared/services/rewards/manager.go
+++ b/shared/services/rewards/manager.go
@@ -52,7 +52,9 @@ func GetClaimStatus(rp *rocketpool.RocketPool, nodeAddress common.Address) (uncl
 
 		var bitmap *big.Int
 		bitmap, err = rp.RocketStorage.GetUint(nil, crypto.Keccak256Hash([]byte("rewards.interval.claimed"), nodeAddress.Bytes(), bucketBytes[:]))
-
+		if err != nil {
+			return
+		}
 		for j := uint64(0); j < 256; j++ {
 			targetIndex := i*256 + j
 			if targetIndex >= currentIndex {

